torch_xla/csrc/convolution.cpp

```diff
@@ -316,6 +318,548 @@ ConvGrads BuildTransposedConvolutionBackward(
 
 }  // namespace
 
+// Convert a PTXLATensorFormat into string. And this code copied/inspired from
+// TF:
+// https://github.com/tensorflow/tensorflow/blob/7f47eaf439d2b81de1aa24b10ed57eabd519dbdb/tensorflow/core/util/tensor_format.cc#L40
+std::string PTXLAToString(PTXLATensorFormat format) {
+  switch (format) {
+    case FORMAT_NHWC:
+      return "NHWC";
+    case FORMAT_NCHW:
+      return "NCHW";
+    case FORMAT_NCHW_VECT_C:
+      return "NCHW_VECT_C";
+    case FORMAT_NHWC_VECT_W:
+      return "NHWC_VECT_W";
+    case FORMAT_HWNC:
+      return "HWNC";
+    case FORMAT_HWCN:
+      return "HWCN";
+    default:
+      LOG(FATAL) << "Invalid Format: " << static_cast<tsl::int32>(format);
+      return "INVALID_FORMAT";
+  }
+}
+
+// Performs some basic checks on PTXLAConvOpAttrs that are true for all kinds of
+// XLA convolutions (as currently implemented). And this code copied/inspired
```

Comment:
you can just say `copied from TF2XLA bridge`.

Reply:
Thanks, Jack, these comments are copied from TF too, do you suggest we delete these comments and only comment `copied from TF2XLA bridge` here? or keep these comments and add `copied from TF2XLA bridge`?